bc_obps/compliance/tests/api/test_compliance_summary.py

```diff
@@ -0,0 +1,194 @@
+from decimal import Decimal
+import pytest
+from django.test import Client
+from model_bakery import baker
+from compliance.models import ComplianceSummary, ComplianceObligation
+
+
+class TestComplianceSummaryEndpoint:
+    endpoint_under_test = "/api/compliance/summaries"
+    client = Client()
+
+    @pytest.fixture
+    def mock_user(self):
+        return baker.make('registration.User')
```

Comment:
We prefer to use recipes instead, they're much more flexible, and avoid us having to declare a string of fixtures just for a few tests by telling the bakers how to mock fields and what other recipe to use if there are foreign key relationships.

For this one, you would only need to instantiate the `report_version` recipe from `reporting/tests/utils/baker_recipes` to get most of your things.
And then let's start another recipes file for the compliance models in the same way